src/app/code/community/Shopgate/Framework/Model/Payment/Simple/Paypal/Standard.php

```diff
@@ -36,7 +36,10 @@ public function manipulateOrderWithPaymentData($magentoOrder)
         $magentoOrder = parent::manipulateOrderWithPaymentData($magentoOrder);
 
         $info        = $this->getShopgateOrder()->getPaymentInfos();
+
+        Mage::dispatchEvent('sales_order_place_before', array(self::TYPE_ORDER => $magentoOrder));
```

Comment:
you may as well plop it in the ```createTransaction()``` calls

Reply:
I didn't want to mess with the return, but it's probably more consistent. done